nbclassic/templates/page.html

```diff
@@ -137,6 +137,26 @@
 </noscript>
 
 <div id="header" role="navigation" aria-label="{% trans %}Top Menu{% endtrans %}">
+  <div  id="newsId" style="display: none">
+    <div class="alert alert-info" role="alert">
+      <div style="display: flex">
+        <div>
+          <span class="label label-warning">NEWS</span>
```

Comment:
Maybe it should say something else than "NEWS" here?

"NEWS" might give the impression there will be new features and updates coming to NBClassic / Notebook 6 in the future, which will not be the case.

Reply:
I have used `UPDATE`